pkgs/development/python-modules/allure-behave/default.nix

```
@@ -9,13 +9,13 @@
 
 buildPythonPackage rec {
   pname = "allure-behave";
-  version = "2.9.45";
+  version = "2.10.0";
 
   disabled = pythonOlder "3.4";
```

Upstream dropped support for [Python < 3.7](https://github.com/allure-framework/allure-python/releases/tag/2.10.0).

```suggestion
  disabled = pythonOlder "3.7";
```